src/ProjectSystemDogfoodSetup/ProjectSystemDogfoodSetup.csproj

```diff
@@ -51,23 +30,16 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.VisualStudio.ProjectSystem.Managed.TestServices\Microsoft.VisualStudio.ProjectSystem.Managed.TestServices.csproj">
-      <Project>{23BCBC90-ED65-4EE3-8AF1-DD7CAEFDBEE9}</Project>
       <Name>Microsoft.VisualStudio.ProjectSystem.Managed.TestServices</Name>
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3bBuiltProjectOutputGroupDependencies%3bGetCopyToOutputDirectoryItems%3bSatelliteDllsProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup%3b</IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
     <ProjectReference Include="..\ProjectSystemSetup\ProjectSystemSetup.csproj">
-      <Project>{9fbd7ef2-9449-486d-9fdd-fa56160aa8bb}</Project>
```

Comment:
I hadn't removed the Name metadata from all VSIX projects because the VSSDK uses that to match the project name from the extension manifest. However these two projects are not mentioned in the manifest so they can be simplified - done.